src/main/java/com/auth0/client/mgmt/ManagementAPI.java

```diff
@@ -364,6 +364,22 @@ public KeysEntity keys() {
         return new KeysEntity(client, baseUrl, tokenProvider);
     }
 
+    /**
+     * Getter for the Prompts Entity
+     * @return the Prompts Entity
+     */
```

Comment:
This and the next one just need the Javadoc fixed to state the correct entity type.

Reply:
Yess, somehow it got missed. I have updated the java doc defining the correct entity